Update duty/refactor

eralpozcan opened the pull request:
- Add onDuty/offDuty
- Add duty check for TakeCarMenu
- Add Config.Location[duty] and [vehicle]
- Remove AddEventHandler and Change RegisterServerEvent > RegisterNetEvent

eralpozcan committed:
Update duty/refactor

- Add onDuty/offDuty
- Add duty check for TakeCarMenu
- Add Config.Location[duty] and [vehicle]
- Remove AddEventHandler and Change RegisterServerEvent > RegisterNetEvent

eralpozcan committed:
Update forgot blip location

Update forgot blip location

eralpozcan committed:
Menu bug basic fix

Issue #6

eralpozcan commented:
Last menu update : It does not close automatically, but it can be closed from outside the vehicle.

GhzGarage commented:
Please get rid of “isLoggedIn” and use the proper state bag method. Everything else looks good!

eralpozcan committed:
Change isLoggedIn to state bag.

Change `isLoggedIn` to `LocalPlayer.state['isLoggedIn']`

Comment line "QBCore:Client:OnPlayerUnload" function.

eralpozcan committed:
Edit takevehicle coords

Config.Location  >  Config.Locations["vehicle"]

eralpozcan committed:
update

update
